xbmc/cores/AudioEngine/Utils/PackerMAT.cpp

```diff
@@ -57,42 +57,27 @@ CPackerMAT::CPackerMAT()
 // high-bitrate streams can overshoot this size and therefor require proper handling of dynamic padding.
 bool CPackerMAT::PackTrueHD(const uint8_t* data, int size)
 {
-  TrueHDMajorSyncInfo info;
-
-  // get the ratebits and output timing from the sync frame
+  // get the ratebits from the sync frame
   if (AV_RB32(data + 4) == FORMAT_MAJOR_SYNC)
   {
-    info = ParseTrueHDMajorSyncHeaders(data, size);
-
-    if (!info.valid)
+    if (size < 32)
       return false;
 
-    m_state.ratebits = info.ratebits;
+    if (data[7] == 0xba)
+      m_state.ratebits = data[8] >> 4;
+    else if (data[7] == 0xbb)
+      m_state.ratebits = data[9] >> 4;
+    else
+      return false;
   }
-  else if (m_state.prevFrametimeValid == false)
+  else if (!m_state.prevFrametimeValid)
   {
     // only start streaming on a major sync frame
-    m_state.numberOfSamplesOffset = 0;
     return false;
   }
 
   const uint16_t frameTime = AV_RB16(data + 2);
   uint32_t spaceSize = 0;
-  const uint16_t frameSamples = 40 << (m_state.ratebits & 7);
-  m_state.outputTiming += frameSamples;
-
-  if (info.outputTimingPresent)
-  {
-    if (m_state.outputTimingValid && (info.outputTiming != m_state.outputTiming))
-    {
-      CLog::Log(LOGWARNING,
```

Comment:
This was quite good I think to track down seek issues - you really don't need it anymore?

Reply:
It was only partly useful: all the problematic streams had discontinuities in output timing. But not all streams with discontinuities had dropouts.

After the latest improvements, no (known) stream with discontinuities has dropouts. 

It has never really been a determining factor to know if the stream had dropouts.

And seek log code is still here:
https://github.com/xbmc/xbmc/blob/24cea35cdd7c54c93f41aaff3b94768ed1de37bf/xbmc/cores/AudioEngine/Utils/PackerMAT.cpp#L96